orc8r/gateway/python/magma/state/state_replicator.py

```diff
@@ -265,3 +264,24 @@ async def _cleanup_deleted_keys(self):
         for key in deleted_keys:
             del self._state_versions[key]
         self._state_keys_from_current_iteration = set()
+
+    # TODO(#8806): update this once we confirm partners no longer set sync_interval service configs.
+    def resolve_sync_interval(self, service):
```

Comment:
if `self` used here? if not, this can be a pure function, not defined in the class

Reply:
also minor: can we add a pyre type hint here?

Reply:
yup can update - wasn't sure what the teams stance on type hints was since i didn't see them elsewhere